### DISCUSSION TITLE
OPENSTEP 4

### DISCUSSION THREAD

**Discussion by gustaf71** [utmapp/UTM#5447] [UTM Help (macOS)]
**Describe the issue**  
How to change floppy image

**Configuration**  
* UTM Version: 4_3_0
* macOS Version: 12_6_6
* Mac Chip (Intel, M1, ...): M1

I use this guide to install OPENSTEP 4
https://openstep.bfx.re

The problem I have in UTM is: **how do I change the floppy image in the setup.**

Regards

**Comment by gustaf71** [comment]
I was able to solve the problem with the floppy.... **I have to enable the checkbox removable.**

<img width="880" alt="image" src="https://github.com/utmapp/UTM/assets/36249438/a5764a8c-c84b-4981-a6ca-c13001fed45d">

**Comment by gustaf71** [comment]
Has anyone try the guide above to install OPENSTEP 4 ??

**Reply by rjhoover** [reply]
I'm trying to but can't get it to boot off the floppy. I really don't know what I'm doing. Could you share how you have all your configuration?

**Reply by tontonraoul** [reply]
I know it's an old subject but no one succeeded in creating an Openstep 4.2 utm ?

**Reply by tifasoftware** [reply]
I had tried NeXTStep 3 and failed, but I don't recall trying 4.2.

**Reply by gustaf71** [reply]
I think the sparc version of OpenStep 4.2 should work. The setup do start but later I get some error.
https://www.youtube.com/watch?v=KlZwLwwgSBE
The problem is the sparc version use SCSI and UTM do not make it simple at least it make it closer to working UTM image.

**Reply by andyfrueh** [reply]
The problem seems to stem from the fact that you
1) can't force a 286 CPU.  The 1.1 and 1.3 patched versions should work regardless of a "fast" 486, but I always end up getting a TRAP 0000 on boot, which usually means it's too fast.
2) you can't force a specific SCSI card, and OpenStep is SUPER picky about that.
3) You can't seem to force 2 IDE drives to be on particular channels or as master/slave - which again, OpenStep is VERY picky about.

I have tried using various machines - ISA only, i440FX...nothing seems to work.

<img width="896" height="343" alt="image" src="https://github.com/user-attachments/assets/93aa51ea-bde0-4f7a-b7f0-5bbe52e36066" />
